jobs/release.Jenkinsfile

```diff
@@ -223,16 +223,19 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                                           credentialsId: 'oscontainer-push-registry-secret')]) {
                         def repo = registry_repos[configname]
                         def (artifact, metajsonname, tag_suffix) = val
-                        def arch_args = basearches.collect{"--arch ${it}"}.join(" ")
-                        def tag_args = " --tag=${params.STREAM}${tag_suffix}"
+                        def extra_args = basearches.collect{"--arch ${it}"}
+                        if (registry_repos.v2s2) {
+                            extra_args += "--v2s2"
+                        }
+                        def tag_args = ["--tag=${params.STREAM}${tag_suffix}"]
                         if (registry_repos.add_build_tag) {
-                            tag_args += " --tag ${params.VERSION}${tag_suffix}"
+                            tag_args += "--tag=${params.VERSION}${tag_suffix}"
                         }
```

Comment:
merely preference here: I would just add these into `extra_args`. I don't think the order of the arguments(options) matters does it?

Reply:
oh I see - we re-use the tag_args below. NVM